### ISSUE TITLE
dataset type manager: no focus after using quick filter

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#45]
After using the filter option in the Dataset Type Manager window, no UI element has focus, so keyboard navigation doesn't work anymore. Note that other places that use this same UI like the Control Strategy Manager and the Control Program Manager don't have this problem.

**Comment by cseppan** [comment]
This looks like the same type of problem as #57; the populate() method in DatasetTypesManagerWindow disables and re-enables all the components in the window. Probably makes sense to have the Filter Fields pull-down menu request focus after the refresh is complete.
